centos.org/pipelines/lib/foremanDuffyJob.groovy

```diff
@@ -0,0 +1,140 @@
+pipeline {
+    agent { label 'cico-workspace' }
+
+    options {
+        timestamps()
+        ansiColor('xterm')
+        buildDiscarder(logRotator(numToKeepStr: '3'))
+    }
+
+    environment {
+        ANSIBLE_CALLBACK_WHITELIST = 'community.general.opentelemetry'
+    }
+
+    stages {
+        stage('Setup Environment') {
+            steps {
+                deleteDir()
+                git url: 'https://github.com/theforeman/forklift.git'
+
+                sh(label: 'pip install', script: 'pip3.8 install --user opentelemetry-api opentelemetry-sdk opentelemetry-exporter-otlp python-cicoclient')
+                sh(label: 'duffy collection install', script: 'ansible-galaxy collection install --collections-path ~/.ansible/collections git+https://github.com/evgeni/evgeni.duffy')
```

Comment:
Will you publish this or do we continue using it from git? Or do you plan to transfer it to CentOS?

Reply:
I talked to arrfab to make this `centos.duffy` at some point.